pkg/disruptors/controller.go

```diff
@@ -133,15 +137,19 @@ func (c *agentController) Visit(ctx context.Context, visitor func(string) []stri
 
 // Targets retrieves the list of target pods for the given PodSelector
```

Comment:
Maybe we could update the comment to refect it now returns the list of the target's names.
```suggestion
// Targets retrieves the names of the target pods for the given PodSelector
```